wonderland/src/index.js

```diff
@@ -38,13 +46,13 @@ WL.registerComponent('zesty-ad', {
     },
 
     start: function() {
-        this.loadAd(this.adSpace, this.creator).then(ad => {
+        this.loadAd(this.adSpace, this.creator, AD_RATIOS[this.defaultAd].type).then(ad => {
             this.ad = ad;
 
             if(this.scaleToRatio) {
               /* Make ad always 1 meter height, adjust width according to ad aspect ratio */
-              this.collision.extents = [0.75*this.height, this.height, 0.1];
-              this.object.scale([0.75*this.height, this.height, 1.0]);
+              this.collision.extents = [AD_RATIOS[this.defaultAd].width * this.height, this.height, 0.1];
+              this.object.scale([AD_RATIOS[this.defaultAd].width * this.height, this.height, 1.0]);
```

Comment:
We should call `this.object.resetScaling()` before, as this will scale additionally to any prior scaling applied to the object.